src/atoms/Toggle/Toggle.native.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState, useContext, useRef } from 'react';
+import { View, Animated, TouchableOpacity } from 'react-native';
+import styled, { ThemeContext } from 'styled-components/native';
+import PropTypes from 'prop-types';
+
+import automationAttributes from '../../_helpers/automation-attributes';
+
+const styles = {
+  size({ size, theme }) {
+    switch (size) {
+      case 'medium':
```

Comment:
Is there only one variation of this component?

Reply:
Yeah, currently there is a medium-size Toggle only, here to accommodate new sizes in the future.